vintf/manifest.xml

```diff
@@ -47,10 +47,11 @@
     <hal format="hidl">
         <name>android.hardware.camera.provider</name>
         <transport>hwbinder</transport>
-        <version>2.4</version>
+        <version>2.5</version>
         <interface>
             <name>ICameraProvider</name>
             <instance>legacy/0</instance>
+            <instance>external/0</instance>
```

Comment:
Is this interface _always_ required? Since this sits in the `common` repo, it might break existing HALs that do not host an `external/0` instance:

https://source.android.com/docs/core/architecture/vintf/match-rules#hidl-and-native-hals

You might have to add a secondary `<hal>` with `optional="true"` to represent this @sraase :cold_sweat:

Reply:
don't we want it included in all SOD devices?

Reply:
Yes, android.hardware.camera.provider@2.5-external-service always hosts that instance.